content/en/docs/concepts/architecture/nodes.md

```diff
@@ -282,16 +281,19 @@ that the scheduler won't place Pods onto unhealthy nodes.
 
 ### Node capacity
 
-The capacity of the node (number of cpus and amount of memory) is part of the node object.
-Normally, nodes register themselves and report their capacity when creating the node object. If
-you are doing [manual node administration](#manual-node-administration), then you need to set node
-capacity when adding a node.
+Node objects include the Node's resource capacity (for example: the amount of memory
+available, and the number of CPUs).
```

Comment:
- This page gets confusing when referring to the Node (object/resource) versus the node (physical machine, virtual machine). I _think_ it is acceptable to refer to the Node (object) as just Node (capitalized).
For instance, see line 116. NodeStatus is a field in the Node object (not the node). Line 12 starts with _Node_ is a worker *machine*. Should this be: a _Node_ represents a physical or virtual machine? Here the sentence is referring to the object not the machine.
- Also check node controller vs NodeController, Node Controller
- Line 284: Nodes maintain information about a node's resource capacity (for example: ...).
- Check lines 286 - 289. I am having trouble parsing this sentence.

Reply:
Thanks for the feedback.

> Line 12 starts with Node is a worker machine. Should this be: a Node represents a physical or virtual machine? Here the sentence is referring to the object not the machine.

I'm OK with the idea that _all_ objects in computing are in some sense an abstraction&mdash;same way that PersistentVolume or Machine (from the cluster API) are also abstract. For that reason I prefer either the existing wording or my proposed rewrite.

Where a sentence starts with “Node&hellip;”, I think it helps to clarify “Node object” because at the start of a sentence Node will be capitalized whatever it refers to/